File: app/src/main/java/com/gasparaiciukas/owntrainer/network/Nutrients.java
package com.gasparaiciukas.owntrainer.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Nutrients {

    @SerializedName("ENERC_KCAL")
    @Expose
    private double calories;
    @SerializedName("CHOCDF")
    @Expose
    private double carbs;
    @SerializedName("FAT")
    @Expose
    private double fat;
    @SerializedName("PROCNT")
    @Expose
    private double protein;

    public double getCalories() {
        return calories;
    }

    public double getCarbs() {
        return carbs;
    }

    public double getFat() {
        return fat;
    }

    public double getProtein() {
        return protein;
    }

}

File: app/src/main/java/com/gasparaiciukas/owntrainer/network/Nutrients.kt
package com.gasparaiciukas.owntrainer.network

import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Nutrients
(
@SerializedName("ENERC_KCAL") @Expose val calories: Double = 0.0,
@SerializedName("CHOCDF") @Expose val carbs: Double = 0.0,
@SerializedName("FAT") @Expose val fat: Double = 0.0,
@SerializedName("PROCNT") @Expose val protein: Double = 0.0
) : Parcelable